Assets/GothicVR/Scripts/Manager/Settings/SettingsManager.cs

```diff
@@ -67,13 +78,16 @@ private string GetRootPath()
                 return Application.streamingAssetsPath;
         }
 
-        private void CheckIfGothicIDirectoryExists()
+        public bool CheckIfGothic1InstallationExists()
         {
-            if (!Directory.Exists(GameSettings.GothicIPath))
-                throw new ArgumentException(
-                    $"GothicI installation path wasn't found at >{GameSettings.GothicIPath}<." +
-                    $"Please put in the right absolute path to your local installation.");
-        }
+            bool isValid = false;
+            if (Directory.Exists(GameSettings.GothicIPath))
+            {
+                var pathGothicExe = GameSettings.GothicIPath + "\\system\\Gothic.exe";
+                isValid = File.Exists(pathGothicExe);
```

Comment:
I think I return a bool. Therefore I used the variable.

Reply:
But what if I want to take only one folder with DATA and _work? That would be good if that also goes.
Because I have modded my original game e.g.

Does not it make more sense to see if the 2 necessary folders are there?

Reply:
Changed it as suggested.